Sources/MapboxMaps/Foundation/MapboxMap.swift

```diff
@@ -706,6 +707,76 @@ extension MapboxMap: MapFeatureQueryable {
                                                                      type: FeatureExtensionValue.self,
                                                                      concreteErrorType: MapError.self))
     }
+
+    /// Returns all the leaves (original points) of a cluster (given its cluster_id) from a GeoJSON source, with pagination support: limit is the number of leaves
+    /// to return (set to Infinity for all points), and offset is the amount of points to skip (for pagination).
+    ///
+    /// - Parameters:
+    ///   - sourceId: The identifier of the source to query.
+    ///   - feature: Feature to look for in the query.
+    ///   - limit: the number of points to return from the query, default to 10
+    ///   - offset: the amount of points to skip, default to 0
+    ///   - completion: The result could be a feature extension value containing
+    ///         either a value (expansion-zoom) or a feature collection (children
+    ///         or leaves). An error is passed if the operation was not successful.
+    public func getGeoJsonClusterLeaves(for sourceId: String,
```

Comment:
```suggestion
    public func getGeoJsonClusterLeaves(forSourceId sourceId: String,
```

The Swift API Design Guidelines recommend including the parameter label to compensate for weak type information (e.g. String is a "weak" type in this context since we don't accept arbitrary strings, only strings that are source IDs.)

Reply:
Similarly below.

Reply:
This will cause error `Instance method 'getGeoJsonClusterLeaves(forSourceId:feature:limit:offset:completion:)' has different argument labels from those required by protocol 'MapFeatureQueryable' ('getGeoJsonClusterLeaves(for:feature:limit:offset:completion:)')`